docs/flox-5-minutes.md

```diff
@@ -5,637 +5,249 @@ description: Get started with creating Flox environments.
 
 # Flox in 5 minutes
 
-Flox is a virtual environment and package manager all in one.
-With Flox you create development environments that put you into reproducible
-subshells with dependencies provided and configured for you.
-Even better, these environments layer so you can prepare different environments
-for different contexts and stack them when needing to work across contexts.
+Flox is a next-generation, language-agnostic package and environment manager.
+With it you create sets of tools, environment variables, and setup scripts that work reproducibly from machine to machine, x86_64 to Arm, and macOS to Linux.
+The cherry on top is that all of this works without causing version conflicts between projects.
 
-Finally, Flox environments are portable across architecture,
-operating system, and the entire software lifecycle.
+We call these stacks **"Flox environments"**.
```

Comment:
I don't think we need both quotes and bold here. My personal preference is to use bold to introduce new terms (less symbol clutter than quotes).

```suggestion
We call these stacks **Flox environments**.
```

Reply:
Fixed